src/test/groovy/life/qbic/datamodel/dtos/business/AffiliationSpec.groovy

```diff
@@ -0,0 +1,28 @@
+package life.qbic.datamodel.dtos.business
+
+import spock.lang.Specification
+
+/**
+ * Simple test class to assert the correct behaviour of the Affiliation Builder.
+ *
+ * @author Sven Fillinger
+ * @since 1.11.0
+ */
+class AffiliationSpec extends Specification{
+
+  def "Fluent API shall create an affiliation object"() {
+    when:
+    Affiliation testAffiliation =
+        new Affiliation.Builder("Universität Tübingen",
+            "Auf der Morgenstelle 10",
+            "72076",
+            "Tübingen").
+            country("Germany").build()
```

Comment:
```suggestion
           .build()
```
Nit: I'd either check if the default setting to "Germany" works or if another country can be defined.